.travis/override.trusty.main

```diff
@@ -0,0 +1,2 @@
+appliance50	optional	cs50
+teamviewer9	optional	cs50
```

Comment:
We have these two packages (`appliance50` and `teamviewer9`) in mirror. This file just overrides a couple of metadata fields in these packages. I don't think we need the packages themselves anymore. Should we delete (and therefore delete this file)?